CONTRIBUTING.md

```diff
@@ -306,9 +313,28 @@ $ ./cli exec --authn-oidc
 root@9feae5e5e001:/src/conjur-server#
 ```
 
+###### Run Cukes with Google Cloud Platform (GCP) Compatible Environment
+
+**Prerequisites**
+- A Google Cloud Platform account. https://cloud.google.com/.
+- Google Cloud SDK installed. https://cloud.google.com/sdk/docs
```

Comment:
Please add a sentence before the link. For example:

- A Google Cloud Platform account. To create an account see https://cloud.google.com/.
- Google Cloud SDK installed. For information on how to install see https://cloud.google.com/sdk/docs